src/worker.js

```diff
@@ -258,7 +272,7 @@ this.onmessage = function(e) {
 #if ASSERTIONS
         } else {
           // else e == 'unwind', and we should fall through here and keep the pthread alive for asynchronous events.
-          out('Pthread 0x' + threadInfoStruct.toString(16) + ' completed its pthread main entry point with an unwind, keeping the pthread worker alive for asynchronous operation.');
+          err('Pthread 0x' + threadInfoStruct.toString(16) + ' completed its pthread main entry point with an unwind, keeping the pthread worker alive for asynchronous operation.');
```

Comment:
```suggestion
          out('Pthread 0x' + threadInfoStruct.toString(16) + ' completed its pthread main entry point with an unwind, keeping the pthread worker alive for asynchronous operation.');
```

Reply:
Since it looks like it's only used for debugging purposes. Correct me if I'm wrong.

Reply:
It is for debugging, yeah. So it actually seems natural to use `err` instead of `out` to me? But I guess we don't have a very clear policy on such logging (which doesn't matter much in practice on the web console). I'm happy to do it either way people prefer.

Reply:
I see. The `out` / `threadPrintOut` however seems to be unused now in `worker.js`, so perhaps these functions can be removed?

Reply:
I guess they can. I added them for completeness, as they may be needed in the future. But I believe we don't DCE this file, so probably best to remove, I'll do that.